src/task2.js

```diff
@@ -0,0 +1,28 @@
+import { csv } from 'csvtojson';
+import { fs } from 'fs';
```

Comment:
and default import here

You can find more details about imports here: https://medium.com/@etherealm/named-export-vs-default-export-in-es6-affb483a0910